src/cmd/root.go

```diff
@@ -22,5 +22,9 @@ func Execute() {
 }
 
 func init() {
+	rootCmd.AddCommand(checkFileCmd)
 	rootCmd.AddCommand(checkCmd)
+
+	checkFileCmd.Flags().Bool("gui-child-mode", false, "Tells app that it is executed by gui")
```

Comment:
I would consider creating variables such as `commandName`, `commandDesc` and `commandDefOption`

Reply:
I don't think it is necessary to create const variable only for holding flag name, default value and description especially that these variables in such case would be used each only once.